iina/AutoFileMatcher.swift

```diff
@@ -45,9 +46,11 @@ class AutoFileMatcher {
 
     // group by extension
     for file in files {
-      let fileInfo = FileInfo(file)
-      if let mediaType = Utility.mediaType(forExtension: fileInfo.ext) {
-        filesGroupedByMediaType[mediaType]!.append(fileInfo)
+      autoreleasepool{
```

Comment:
Putting `autoreleasepool` inside loop is also a common practice to avoid memory spike, if that's what you asking.